lib/web_ui/test/canvaskit/embedded_views_test.dart

```diff
@@ -241,26 +244,20 @@ void testMain() {
       //   Expect: main canvas plus platform view overlays; empty cache.
```

Comment:
Here and below I think the docs for expectations should now be updated, e.g. that we no longer expect empty cache.

Reply:
Done.